### ISSUE TITLE
TPAC 2022 status report

### ISSUE THREAD

**Issue by LJWatson** [w3c/IndexedDB#389]
[TPAC 2022 ](https://www.w3.org/2022/09/TPAC/) is hybrid this year. WebApps will meet in Vancouver on [12 and 13 September](https://www.w3.org/calendar/tpac2022/). Please [register](https://www.w3.org/2022/09/TPAC/registration.html) if you plan to attend in person or remotely.

Editors, we need you to do two things before 30 August 2022:

1. If you want agenda time to discuss your spec(s), please use W3C/webappswg#87 to request it
2. Complete a status report for your spec(s)

#### Status report

Your status report should answer these questions (as a comment on this issue):

1. What progress has your spec made in the last 12 months?
2. Is anything blocking your spec from moving to CR/Rec?
3. If yes, what is your plan to unblock it and do you need any help?

Please post a status report on your spec(s) even if you do not plan to attend TPAC at all.

CC @marcoscaceres and @siusin

**Comment by inexorabletash** [comment]
I'll get to this soon.

I don't think I'll have an agenda (but welcome any topics anyone else wants to propose). I will ask for the chairs' help in rallying support for an additional active editor on the spec from among the other implementers.

**Comment by inexorabletash** [comment]
For reference:

TPAC 2020 #337 
TPAC 2021 #364

# 2022 Status Report

## Progress in the last 12 months

Same as last time, unfortunately - extremely limited progress. 

Normative changes:
* #383 

Editorial updates:
* #384
* #387 
* #388

Implementer updates:

* Chromium focus has been on third party storage partitioning, prototyping storage buckets, and file system access handles, so not much work has happened with Indexed DB specifically. No additional prototyping on new methods has been done since last year.
* Safari 15 supports IDBCursor.prototype.request, IDBTransaction.prototype.commit(), IDBTransaction.prototype.durability, IDBTransactionOptions durability member. This means all the new v3 members are now supported in all 3 engines. 
    *  TODO: update the 🚧 ...new in this edition. It is supported in ... 🚧 banners!

Active PRs:

* #386 (for #375)
* #334 

Expected near future work:

* Integration with Storage Buckets
* #381 

## Moving to CR/Rec

* FPWD #348 from last year. Progress!
* As noted last year, the actual normative deltas since V2 are fairly minimal. That said, they're now adopted by all 3 engines, so publishing a V3 as soon as possible makes sense.
* However, the big needs for the spec are around integration with other specification infrastructure. Until these are sorted out, it feels inappropriate to move forward. They are:
   * Back/forward cache
   * Storage endpoints
   * Storage buckets

## What is your plan to unblock it and do you need any help?

* Active editors from other implementors would be of significant benefit.
    * Even simple issues like #386 are stalled without being able to get input from other browsers. 
    * Brainstorming on #334 in particular would be welcome, and my hope is to discuss next steps with anyone interested at TPAC.

**Comment by inexorabletash** [comment]
Correction: Firefox does not support the IDBFactory `databases()` method, the IDBTransaction `durability()` method or IDBTransactionOptions `durability` option. My bad.

**Comment by marcoscaceres** [comment]
Thanks again for preparing this last year @inexorabletash. Closing in prep for TPAC 2023.
